src/main/java/frc/robot/RobotMap.java

```diff
@@ -89,4 +342,168 @@ private void setupDrivetrain()
 
     drivetrain = new MecanumDrive(frontLeftDriveTalon, rearLeftDriveTalon, frontRightDriveTalon, rearRightDriveTalon);
   }
+
+  /**
+   * This method instantiates the arm talons and sets their parameters
+   */
+  private static void setupArm()
+  {
+    firstArmSegmentLeftTalon = new WPI_TalonSRX(FIRST_ARM_SEGMENT_LEFT_TALON_ID);
+    firstArmSegmentRightTalon = new WPI_TalonSRX(FIRST_ARM_SEGMENT_RIGHT_TALON_ID);
+    secondArmSegmentLeftTalon = new WPI_TalonSRX(SECOND_ARM_SEGMENT_LEFT_TALON_ID);
+    secondArmSegmentRightTalon = new WPI_TalonSRX(SECOND_ARM_SEGMENT_RIGHT_TALON_ID);
+
+    wristTalon = new WPI_TalonSRX(WRIST_TALON_ID);
+    gimbalTalon = new WPI_TalonSRX(GIMBAL_TALON_ID);
+
+    //Need to add encoders to all talons 
```

Comment:
Most editors support detection of TODO in comments, you should add it to comments like this for visibility

Reply:
Oh ok. Will do